portfolio/src/main/webapp/index.html

```diff
@@ -85,15 +88,12 @@ <h4 class="category">-INTERESTS & HOBBIES-</h4>
 
     <div>
         <button onclick="scrollUp()" id="scroll-button"><img src="images/up.png" alt="up arrow"></img></button>
-
-        <a href="https://github.com/sunr1" target="_blank"><img src="https://img.icons8.com/ios-filled/50/000000/github.png" alt="images/linkedin.png"/></a>
-        <a href="https://linkedin.com/in/rebecca-sun/" target="_blank"><img src="https://img.icons8.com/color/48/000000/linkedin.png" alt="images/github.png"/></a>
-        <a href="mailto:rebeccasun@google.com" target="_blank"><img src="https://img.icons8.com/fluent/48/000000/gmail.png" alt="images/gmail.png"/></a>
     </div>
 
     <footer>
         <p id="contact">Get in touch: <a href="mailto:rebeccasun@google.com" target="_blank"> rebeccasun@google.com</a></p>
         <p id="credits">Created and designed by Rebecca Sun</p>
+        <button onclick="confetti()"></button>
```

Comment:
Is this button intentional?  Like an easter egg haha?  The confetti effect is super cool!  But it shows up as a tiny sliver across the very bottom of the webpage.

Reply:
I made it a little bigger haha thanks for pointing that out